external/RSL_LITE/period.c

```diff
@@ -169,12 +169,12 @@ void RSL_LITE_PACK_PERIOD ( int* Fcomm0, char * buf , int * shw0 , int * typesiz
 	  MPI_Abort(MPI_COMM_WORLD, 98) ;
         }
         if ( typesize == 8 ) {
-          F_PACK_LINT ( buf, p+xp_curs, imemord, &js, &je, &ks, &ke, &is, &ie,
+          F_PACK_LINT ( (long *)buf, (long *)p+xp_curs, imemord, &js, &je, &ks, &ke, &is, &ie,
```

Comment:
Although I'm not certain, I suspect we need parentheses here, with
`(long *)p+xp_curs` becoming `(long *)(p+xp_curs)`.

Reply:
I think that's correct. I had that issue with the `F_[UN]PACK_INT` commands, but I don't think our tests exercise this section and evidently I forgot to update this section in those particular fixes.